### ISSUE TITLE
Browser specific delays for tests

### ISSUE THREAD

**Issue by Perni1984** [wKich/creevey#174]
I am running into timing issues when doing the screenshots especially with webkit compared to firefox and chrome. The `delay` parameter somewhat brings relief to this, but when using the `delay` for a test it would slow down all the tests in all other browsers aswell.

We are testing mobile/tablet/desktop resolutions in chrome/firefox/webkit - so we have 9 different versions of the same test. The delay (e.g. 2000 ms) would be a total delay of 18.000 ms (9 x 2.000ms) for us, which adds up significantly for our testing pipeline.

Would it be possible to delay the test only for specific browsers and introducing a syntax comparable to the `skip` parameter. E.g.

```
 parameters: {
    creevey: {
      delay: {
        in: ["webkit.mobile", "webkit.tablet", "webkit.desktop"],
        ms: 2000
      }
    }
  }
```

**Comment by wKich** [comment]
Yeah, it's possible to make it happen. I'll try to add this feature in the next week

**Comment by Perni1984** [comment]
thats great @wKich - many thanks!
